### DISCUSSION TITLE
Totum-AI

### DISCUSSION THREAD

**Discussion by dmbrtlt** [totumonline/totum-ru-issues-and-discussions#626] [Как это сделать?] [answered]
Добрый день, вроде в середине октября писали про запуск Totum-AI, а где можно про него информацию найти?

**Comment by ttm-alexeypolunin** [comment]
Модель, с которой мы будем запускаться, сейчас в обучении. Эстимейтед тайм у нее завтра. Так что я думаю в среду уже все будет работать.

**Reply by ttm-alexeypolunin** [reply, marked as answer]
https://ru.totum.online/ai
